.github/workflows/publish_docker.yaml

```diff
@@ -86,13 +65,52 @@ jobs:
           repository: ethyca/fidesinfra
           token: ${{ secrets.DISPATCH_ACCESS_TOKEN }}
 
+  Push:
```

Comment:
@adamsachs I'll definitely need your help checking this logic, also thinking we need some way to test this if possible but maybe it isn't possible 🤷